boards.txt

```diff
@@ -3619,140 +3631,6 @@ um_pros3.menu.EraseFlash.all.upload.erase_cmd=-e
 
 ##############################################################
 
-um_rmp.name=UM RMP
-um_rmp.vid.0=0x303a
-um_rmp.pid.0=0x80F6
-
-um_rmp.upload.tool=esptool_py
-um_rmp.upload.tool.default=esptool_py
-um_rmp.upload.tool.network=esp_ota
-
-um_rmp.upload.maximum_size=1310720
-um_rmp.upload.maximum_data_size=327680
-um_rmp.upload.flags=
-um_rmp.upload.extra_flags=
-um_rmp.upload.use_1200bps_touch=true
-um_rmp.upload.wait_for_upload_port=true
-
-um_rmp.serial.disableDTR=false
-um_rmp.serial.disableRTS=false
-
-um_rmp.build.tarch=xtensa
-um_rmp.build.bootloader_addr=0x1000
-um_rmp.build.target=esp32s2
-um_rmp.build.mcu=esp32s2
-um_rmp.build.core=esp32
-um_rmp.build.variant=um_rmp
```

Comment:
does this have a variant folder that needs removing?

Reply:
Oh, whoops, sorry, forgot to commit that, hang on :(